accord-core/src/test/java/accord/impl/list/ListRead.java

```diff
@@ -78,13 +78,17 @@ public AsyncChain<Data> read(Seekable key, SafeCommandStore safeStore, Timestamp
         {
             default: throw new AssertionError();
             case Key:
+                if (!keys.contains((Key)key))
```

Comment:
the main reason bootstrap regression failed to be detected in burn test was that it switched to using the sync point's emptyTxn, which has 0 keys... since ListRead supports ranges it was able to return the data requested from bootstrap even though its inputs were outside of the range of `keys`... This check helps protect that from happening again.